content/docs/quickstart/_index.md

```diff
@@ -7,6 +7,30 @@ weight: 1
 hidechildren: true # this flag hides all sub pages in the sidebar-multicard.html
 ---
 
+## Keptn Hello World
+
+Run the Keptn Hello, World! example:
+
+```
+docker run --rm -it \
+--name thekindkeptn \
+-v /var/run/docker.sock:/var/run/docker.sock:ro \
+gardnera/thekindkeptn:0.0.5
+```
+
+The Keptn Bridge (UI) will be available on `http://localhost`
+
+This will:
+
+- Create a Kubernetes cluster
+- Install Keptn and remove all authentication (for demo purposes)
+- Install and authenticate the `keptn` CLI
```

Comment:
Should this include either commands that the user can just copy/paste or at least a link to instructions?  The process is documented in https://keptn.sh/docs/0.13.x/reference/cli/

Reply:
No, that's what the docker run command does under the hood. The only command required is the `docker run` one.